modules/tinymce/src/core/main/ts/api/dom/ControlSelection.ts

```diff
@@ -439,11 +439,11 @@ const ControlSelection = (selection: EditorSelection, editor: Editor): ControlSe
     });
 
     controlElm = e.type === 'mousedown' ? e.target : selection.getNode();
-    controlElm = SelectorFind.closest(SugarElement.fromDom(controlElm), 'table,img,figure.image,hr,video,span.mce-preview-object').getOrUndefined()?.dom;
+    controlElm = SelectorFind.closest<HTMLElement>(SugarElement.fromDom(controlElm), 'table,img,figure.image,hr,video,span.mce-preview-object').getOrUndefined()?.dom;
 
     if (isChildOrEqual(controlElm, rootElement)) {
       disableGeckoResize();
-      startElm = selection.getStart(true);
+      startElm = selection.getStart(true) as HTMLElement;
```

Comment:
This doesn't look right. How do we know for certain that this is a HTML element? I think the main problem is the type on `startElm` should be `Element` instead.

Reply:
The problem I was having is when I change it to `Element` it cascades into lots of other type errors. It might be better to just revert the type changes and we can fix it properly later

Reply:
I tested it locally before commenting and it doesn't cause any issues for me? I didn't try `controlElm` at the time though and that does cause type errors so for now I'd suggest just changing the type for `startElm`.

Reply:
Thanks. I have done that